Client/Clique/Modules/CreatedEvents/CreatedEventsViewController.swift

```diff
@@ -8,7 +8,72 @@
 import UIKit
 
 class CreatedEventsViewController: UIViewController {
+    
+    @IBOutlet private var _tableView : UITableView?
+    
+    private var _events : [Event] = [] {
+        didSet {
+            _tableView?.reloadData()
+        }
+    }
+    private var _eventServices = EventServices()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupUI()
+        _getEvents()
+    }
+}
+
+private extension CreatedEventsViewController {
+    func setupUI(){
+        _tableView?.delegate = self
+        _tableView?.dataSource = self
+        _tableView!.register(UINib(nibName: Constants.Storyboards.eventListTableViewCell, bundle: nil), forCellReuseIdentifier: Constants.Storyboards.eventListTableViewCell)
```

Comment:
Don't use force-unwrap in this part of the code. Potentially, this can crash the app.

Also, the force-unwrap (!) won't be needed once you update the outlet.